applications/nrf_desktop/src/modules/hid_forward.c

```diff
@@ -35,14 +39,15 @@ struct hids_subscriber {
 	struct bt_gatt_hids_c hidc;
 	uint16_t pid;
 	uint8_t hwid[HWID_LEN];
+	struct k_delayed_work read_rsp;
+	struct config_event *cfg_chan_rsp;
+	uint8_t cur_poll_cnt;
```

Comment:
It will not help with saving memory (in both cases we have 168 bytes).
If we want to actually save memory, we would have to move `read_rsp` and `cfg_chan_rsp` above `pid` - then we save 8 bytes